.github/workflows/php-coding-standards.yml

```diff
@@ -0,0 +1,44 @@
+name: PHP Coding Standards
+
+on:
+  push:
+    branches:
+      - "trunk"
+    paths:
+      - "**.php"
+      - .github/workflows/php-coding-standards.yml
+  pull_request:
+    paths:
+      - "**.php"
+      - .github/workflows/php-coding-standards.yml
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  phpcs:
+    name: PHP coding standards
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
```

Comment:
💅 Just curious can we use checkout@v4 (using Node.js v20 runtime)?

Reply:
Considering the current .nvmrc is still `lts/fermium` (v14), I plan to update it along with the work for #108.